docs/index/index-management.md

```diff
@@ -78,7 +78,7 @@ We do recommend you to use this management for index.
 
 ### Manual Refresh
 
-When user creates a index specifying manual refresh semantic, the index is created with status *disabled* and query will NOT use this index until user can issue REFRESH INDEX command to build the index. For every REFRESH INDEX command, system will trigger a full rebuild of the index. After rebuild is done, system will change index status to *enabled*, so that it can be used in query rewrite.
+When user creates a index specifying manual refresh semantic, the index is created with status *disabled* and query will NOT use this index until user can issue REFRESH INDEX command to build the index. For every REFRESH INDEX command, system will trigger a full refresh of the index. After a refresh is done, system will change index status to *enabled*, so that it can be used in query rewrite.
```

Comment:
` After a refresh is done` to `Once the refresh operation is finished`

Reply:
done